File: benchmarkDS/Vanilla/CLAUDE_claude-sonnet-4-5-20250929/Lse23.py

import random

def generate_and_print_random_integers():
    # Generate three random integers larger than 1000
    a = random.randint(1001, 10000)
    b = random.randint(1001, 10000)
    c = random.randint(1001, 10000)
    
    # Print the three integers
    print(f"a : {a}")
    print(f"b : {b}")
    print(f"c : {c}")
    
    # Add the three integers and print the result
    sum_result = a + b + c
    print(f"sum: {sum_result}")
    
    # Print sum2 (alternative calculation)
    sum2 = a + b + c
    print(f"sum2: {sum2}")

if __name__ == "__main__":
    print("Test Case 1:")
    generate_and_print_random_integers()
    print()
    
    print("Test Case 2:")
    generate_and_print_random_integers()
    print()
    
    print("Test Case 3:")
    generate_and_print_random_integers()
    print()
    
    print("Test Case 4:")
    generate_and_print_random_integers()
    print()
    
    print("Test Case 5:")
    generate_and_print_random_integers()

File: benchmarkDS/Vanilla/CLAUDE_claude-sonnet-4-5-20250929/Lse23.c

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

void generateAndPrintRandomIntegers() {
    // Generate three random integers larger than 1000
    int a = rand() % 9000 + 1001; // generates numbers from 1001 to 10000
    int b = rand() % 9000 + 1001;
    int c = rand() % 9000 + 1001;
    
    // Print the three integers
    printf("a : %d\\n", a);
    printf("b : %d\\n", b);
    printf("c : %d\\n", c);
    
    // Add the three integers and print the result
    int sum = a + b + c;
    printf("sum: %d\\n", sum);
    
    // Print sum2 (alternative calculation)
    int sum2 = a + b + c;
    printf("sum2: %d\\n", sum2);
}

int main() {
    srand(time(NULL)); // Seed the random number generator
    
    printf("Test Case 1:\\n");
    generateAndPrintRandomIntegers();
    printf("\\n");
    
    printf("Test Case 2:\\n");
    generateAndPrintRandomIntegers();
    printf("\\n");
    
    printf("Test Case 3:\\n");
    generateAndPrintRandomIntegers();
    printf("\\n");
    
    printf("Test Case 4:\\n");
    generateAndPrintRandomIntegers();
    printf("\\n");
    
    printf("Test Case 5:\\n");
    generateAndPrintRandomIntegers();
    
    return 0;
}
